modules/ossm-validating-smcp.adoc

```diff
@@ -32,28 +32,28 @@ You view the Jaeger components under the {JaegerName} Operator and the Elasticse
 You can verify the {SMProductShortName} control plane installation in the {product-title} web console.
 
 . Navigate to *Operators* -> *Installed Operators*.
-. Select the `<istio-system>` namespace.
+. Select the `istio-system` namespace.
 . Select the {SMProductName} Operator.
 .. Click the *Istio Service Mesh Control Plane* tab.
 .. Click the name of your control plane, for example `basic`.
 .. To view the resources created by the deployment, click the *Resources* tab. You can use the filter to narrow your view, for example, to check that all the *Pods* have a status of `running`.
 .. If the SMCP status indicates any problems, check the `status:` output in the YAML file for more information.
-.. Navigate back to *Operators* -> *Installed Operators*.
 
+. Navigate back to *Operators* -> *Installed Operators*.
```

Comment:
PEER REVIEWER I learned recently that we should avoid _Navigate_ when describing the web console. However, the word is used throughout the assembly.